tests/conftest.py

```diff
@@ -119,14 +120,15 @@ def android_emu_driver(request, data_center):
 
     username_cap = environ['SAUCE_USERNAME']
     access_key_cap = environ['SAUCE_ACCESS_KEY']
+    release_version = ReleaseVersion.latest_react_native_github_release()
 
     caps = {
         'username': username_cap,
         'accessKey': access_key_cap,
         'deviceName': 'Android GoogleAPI Emulator',
         'platformVersion': '10.0',
         'platformName': 'Android',
-        'app': "https://github.com/sentry-demos/sentry_react_native/releases/download/1.9/app-release.apk",
+        'app': f'https://github.com/sentry-demos/sentry_react_native/releases/download/{release_version}/app-release.apk',
```

Comment:
I didn't know Python 3 has these f'strings, as they're calling them. Good find.

It did trick me up at first, it reminded me of unicode strings in python like `ustring = u'A unicode \u018e string \xf1'` which is valid for writing in python.